tests/atest/rules/ignored-data/test.robot

```diff
@@ -0,0 +1,11 @@
+# robocop: disable=1001
```

Comment:
you can also create more robot files inside "ignored-data" directory, it scan whole directory. For example happy-path.robot (without ignored data), and the one without # robocop: disable line

Reply:
Agree, I added more test scenarios. ;)